terraform/modules/check-user-expiry-lambda/s3_objects/email_body.txt.j2

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+<meta content="telephone=no" name="format-detection"> <!-- need to add formatting for real phone numbers -->
+<title>Page title</title>
+<style>
+@media only screen and (min-device-width: 581px) {
+.content {
+width: 580px !important;
+}
+}
+</style>
+</head>
+<body style="font-family: Helvetica, Arial, sans-serif;font-size: 16px;margin: 0;color:#0b0c0c;">
+<!--[if (gte mso 9)|(IE)]>
+<table width="580" align="center" cellpadding="0" cellspacing="0" border="0">
+<tr>
+<td>
+<![endif]-->
+<table width="100%" cellpadding="0" cellspacing="0" border="0">
+<tr>
+<td width="100%" height="53" bgcolor="#0B0C0C">
+<table width="580" cellpadding="0" cellspacing="0" border="0" align="center">
+<tr>
+<td width="70" bgcolor="#0B0C0C" valign="middle">
+<!-- This asset should be hosted by the service -->
+<a href="https://www.gov.uk" style="text-decoration: none;"><img src="https://raw.githubusercontent.com/alphagov/email-template/master/crown-32px.gif" alt="" height="32" border="0"></a>
+</td>
+<td width="100%" bgcolor="#0B0C0C" valign="middle" align="left">
+<span style="padding-left: 5px;"><a href="https://www.gov.uk" style="font-family: Helvetica, Arial, sans-serif; font-size: 28px; line-height: 1.315789474; font-weight: 700; color: #EFEFEF; text-decoration: none;">GOV.UK</a></span></a>
+</td>
+</tr>
+</table>
+</td>
+</tr>
+</table>
+<table class="content" align="center" cellpadding="0" cellspacing="0" border="0" style="width: 100%; max-width: 580px;">
+<tr>
+<td>
+<h1 style="font-family: Helvetica, Arial, sans-serif; font-size: 36px; line-height: 1.315789474; font-weight: 700; margin: 19px 0 38px 0;">
+DataWorks AWS Access Management
+</h1>
+<p style="font-family: Helvetica, Arial, sans-serif; font-size: 19px; line-height: 1.315789474; margin: 30px 0 30px 0;">
+Hi [[ username ]],
+</p>
+<p style="font-family: Helvetica, Arial, sans-serif; font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">
```

Comment:
We need to ensure that the secret version of this email is different to this template to prevent phishing.